presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java

```diff
@@ -418,5 +425,25 @@ private synchronized void updateMemoryUsage(TaskStatus taskStatus)
             previousMemory = currentMemory;
             stateMachine.updateMemoryUsage(deltaMemoryInBytes);
         }
+
+        private ExecutionFailureInfo identifyHostGone(ExecutionFailureInfo executionFailureInfo)
```

Comment:
@raghavsethi  I think the most accurate names are
- `checkHostStatusToRewriteFailureInfo`
- `rewriteFailureInfoBasedOnHostStatus`

... So I think the best decision is to use `checkHostStatus`